src/plugins/plot/MctPlot.vue

```diff
@@ -269,13 +278,14 @@ export default {
         }
     },
     data() {
+        this.yAxisIdVisibility = {};
```

Comment:
please move to return block.

Reply:
Hmm. Doesn't that make it reactive? We don't need it to be.